tests/org.jboss.tools.websockets.ui.test/src/org/jboss/tools/websockets/ui/test/WebsocketsUIAllTests.java

```diff
@@ -0,0 +1,28 @@
+/******************************************************************************* 
```

Comment:
I did not use TestSuite in the JAX-RS test plugin. IMO, it's not necessary when all test should be run anyway. It avoids the risk of forgetting to include tests in the suite.